oam/install_scripts/pre-uninstall

```diff
@@ -37,64 +34,61 @@ done
 $installdir/bin/columnstore stop > /dev/null 2>&1
 $installdir/myql/columnstore-Mysql stop > /dev/null 2>&1
 
-# uninstall OS scripts updated by postConfigure
-if test -f /etc/exports.columnstoreSave ; then
-	$SUDO mv -f /etc/exports.columnstoreSave /etc/exports > /dev/null 2>&1
-else
-	$SUDO rm -f /etc/exports > /dev/null 2>&1
+if [ $user != "root" ]; then
+	export COLUMNSTORE_INSTALL_DIR=$installdir
+	export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$installdir/lib:$installdir/mysql/lib
 fi
 
-export COLUMNSTORE_INSTALL_DIR=$installdir
-
-cloud=`$COLUMNSTORE_INSTALL_DIR/bin/getConfig Installation Cloud`
+cloud=`$installdir/bin/getConfig Installation Cloud`
 if [ $cloud = "amazon-ec2" ] || [ $cloud = "amazon-vpc" ]; then
 	if test -f /etc/fstab ; then
-		$SUDO sed -i '/Columnstore\/data/d' /etc/fstab > /dev/null 2>&1
+	    sed -i '/Columnstore\/data/d' /etc/fstab > /dev/null 2>&1
+	fi
+fi
+
+#remove profile file
+if [ $user != "root" ]; then
+	if [ -f $prefix/.bash_profile ]; then		
+		profileFile=$prefix/.bash_profile
+	elif [ -f $prefix/.profile ]; then		
+		profileFile=$prefix/.profile
+	else
+		profileFile=$prefix/.bashrc
+		touch $prefix/.bashrc
+	fi
+
+	if [ -f ${profileFile}_backup ] ; then
+	    mv -f ${profileFile}_backup $profileFile
```

Comment:
This would clobber changes they've made since then.  But also I suggest we not mod their login scripts.